lib/Doctrine/DBAL/Types/Type.php

```diff
@@ -245,7 +245,12 @@ public function getBindingType()
      */
     public static function getTypesMap()
     {
-        return self::$_typesMap;
+        return array_map(
+            static function (Type $type) : string {
+                return get_class($type);
+            },
```

Comment:
why not directly `array_map('get_class', self::getTypeRegistry()->getMap())`? Can't SA tools figure this out? Or is it because `get_class` might return `false`?

Reply:
I personally always avoid string callbacks for these functions. It can't return false in this context.

Reply:
ok then